res/devices/devices.json

```diff
@@ -4,8 +4,16 @@
     "profileName": "canprofile",
     "description": "Example can device",
     "labels": [ "can", "bus" ],
-    "protocols": {
-      "Counter": { "Index": "3" }
+    "protocols": 
+    {
+            "Address":
```

Comment:
according to the device service design convention, the first level of protocol properties is normally the name of the protocol with lowercase, so I suggest use `can` instead of `Address`